workflow.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: nerdalize.com/v1alpha1
```

Comment:
I think we should rename the file to workflow-example.yaml. Or in a subdir called examples/